flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -278,4 +338,31 @@ public void alterPartitionColumnStatistics(
       throws PartitionNotExistException, CatalogException {
     throw new UnsupportedOperationException();
   }
+
+  @VisibleForTesting
+  static SchemaChange[] getSchemaChange(CatalogDatabase current, CatalogDatabase updated) {
+    Map<String, String> currentProperties = current.getProperties();
```

Comment:
What if comment changed?

Reply:
@FANNG1 I think it is not supported to change the schema comment in Gravitino. https://github.com/datastrato/gravitino/blob/main/api/src/main/java/com/datastrato/gravitino/SchemaChange.java#L28
So does the Flink. https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/dev/table/sql/alter/#alter-database